iwyu.cc

```diff
@@ -2201,24 +2201,55 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
   // an iwyu warning now, even without knowing the exact overload.
   // In that case, we store the fact we warned, so we won't warn again
   // when the template is instantiated.
-  // TODO(csilvers): to be really correct, we should report *every*
-  // overload that callers couldn't match via ADL.
+  // Otherwise, all directly '#include'd overloads are reported just to
```

Comment:
Could you just say `included` instead of `'#include'd` -- easier to read, and doesn't interfere with `#include` grepping.